tests/config/pytest_modules.yml

```diff
@@ -1074,9 +1074,6 @@ hmmer/hmmfetch:
 hmmer/hmmsearch:
```

Comment:
I don't seem to be able to edit the code on your branch, needs hmmer/hmmsearch removing from here to be able to update the branch to master.